### PR TITLE
[TC-2051] Fix Spree 4.0 changes

### PR DESCRIPTION
**Pull request by erik-terracycle**

### CODE REVIEW

File: spec/controllers/spree/admin/stores_controller_spec.rb [hunk 1]
```diff
@@ -1,24 +1,17 @@
-require 'spec_helper'
+equire 'spec_helper'
```
Comment (SkayHUN) [inline spec/controllers/spree/admin/stores_controller_spec.rb]: typo

File: spec/controllers/spree/admin/stores_controller_spec.rb [hunk 2]
```diff
@@ -1,24 +1,17 @@
...
-    it "renders index" do
-      spree_get :index
-      response.should be_success
-    end
-  end
+  describe 'on :index' do
+    it 'renders index' do
+      get :index
 
-  context 'update' do
```
Comment (SkayHUN) [inline spec/controllers/spree/admin/stores_controller_spec.rb:17]: did we remove this part intentionally?

**Review by SkayHUN — CHANGES_REQUESTED** [review]
check comments

File: spec/controllers/spree/admin/stores_controller_spec.rb (hunk 2 above)
Comment (erik-terracycle) [reply spec/controllers/spree/admin/stores_controller_spec.rb:17]: yes

**Review by SkayHUN — APPROVED** [review]

**Review by adamFoldvari — APPROVED** [review]

**Review by AHolcsik — APPROVED** [review]

**Review by VasiliyMaksymov — APPROVED** [review]

**Review by p0lish — APPROVED** [review]

**Review by storhet — APPROVED** [review]

**Review by bluta — APPROVED** [review]

**Pull request merged**